### PR TITLE
Add a script to build the wheel

### PR DESCRIPTION
**Pull request by riedgar-ms**
Adds a simple script to build the AdaTest wheel. This takes care of building the TypeScript interface and creating the `.whl` file.

Also tweak the `setup.py`:

- Sort the `install_requires`
- Add `cryptography` to the `install_requires`
- blacken the file

### CODE REVIEW

**Comment by riedgar-ms** [comment]
If this approach is agreeable, then an extra ToDo is to add a build which runs the script and tries to install the wheel produced.

**Comment by riedgar-ms** [comment]
Ping @slundberg @Harsha-Nori

File: scripts/build_wheel.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+def build_client():
+    # Find our initial directory
+    _logger.info("Starting build_client")
+    _logger.info("Running npm install")
+    subprocess.check_call(
+        ['npm', 'install'],
+        cwd='client',
+        shell=True
+    )
```
Comment (Harsha-Nori) [inline scripts/build_wheel.py:16]: Might just be user error on my side, but I run into this error when I hit this code running the script from my adatest base repository:

```bash
npm@8.3.1 /home/harsha/.nvm/versions/node/v16.14.0/lib/node_modules/npm
Traceback (most recent call last):
  File "scripts/build_wheel.py", line 41, in <module>
    main()
  File "scripts/build_wheel.py", line 33, in main
    build_client()
  File "scripts/build_wheel.py", line 12, in build_client
    subprocess.check_call(
  File "/home/harsha/miniconda3/envs/adatest-dev/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
```

`cd client && npm install` works. Am I doing something wrong?

File: setup.py [hunk 2]
```diff
@@ -4,31 +4,42 @@
...
+        "appdirs",
+        "cryptography",
+        "diskcache",
+        "nest_asyncio",
+        "numpy",
+        "pandas",
+        "profanity",
+        "scikit-learn",
+        "shap",
+    ],
```
Comment (Harsha-Nori) [inline setup.py:44]: I assume all these changes are just black or some other formatter, right?

File: scripts/build_wheel.py (hunk 1 above)
Comment (riedgar-ms) [reply scripts/build_wheel.py:16]: The `cwd` argument to `check_call()` is supposed to do that :-/

File: setup.py (hunk 2 above)
Comment (riedgar-ms) [reply setup.py:44]: black, plus a manual sort of the installs. Which now I look at it, wasn't quite right.

File: scripts/build_wheel.py (hunk 1 above)
Comment (riedgar-ms) [reply scripts/build_wheel.py:16]: Would be helpful if `subprocess` printed out the actual error, though.

I take it that just running `npm install` yourself worked fine?

Comment (Harsha-Nori) [reply scripts/build_wheel.py:16]: Yeah, running `cd client && npm install` works fine from root

Comment (riedgar-ms) [reply scripts/build_wheel.py]: Hmmm.... just reread the `subprocess` docs a little more closely, and have pushed a change. Does that give a more informative error?

**Review by Harsha-Nori — APPROVED** [review]
Working on Powershell + WSL -- thanks for the contribution!

**Pull request merged**